include/convex_bodies/orderpolytope.h

```diff
@@ -188,11 +180,11 @@ class OrderPolytope {
             // DON'T JUST check violation of point between 0 and 1
             // as b will change for shifted polytope
             diff = -pt_coeffs(i) - b(i);
-            if (_normalized)    diff /= _row_norms(i);
+            // if (_normalized)    diff /= _row_norms(i); // row_norm is 1
```

Comment:
is this needed as a comment? If not please delete.

Reply:
removed 👍🏽